### DISCUSSION TITLE
Welcome to Elixir LangChain Discussions!

### DISCUSSION THREAD

**Discussion by brainlid** [brainlid/langchain#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do, or intend to do, with LangChain in your Elixir projects.

**Comment by slashmili** [comment]
Hey 👋🏼. I'm Milad, an avid listener of ThinkingElixir and have been following the LangChain library since it was first introduced on the podcast 😉

As of today, I don't have any practical use for LangChain, I'm just learning about Machine Learning and LLMs, especially through attending deeplearning.ai courses. At the same time, I'm attempting to do the course exercises in Elixir (though I fail most of the time).

I'm also scheduled to [talk about RAG](https://www.elixirconf.eu/talks/nearest-neighbors-distant-words-unleashing-the-power-of-ann-in-llms/) in 2 weeks at ElixirConf EU. I was hoping to contribute some RAG-related libraries back to LangChain as part of the preparation for the talk, but unfortunately, I was not able to do so.

**Reply by brainlid** [reply]
Hi @slashmili! Sorry to hear you couldn't get to the point you wanted with RAGs in time for your presentation. Good luck on your talk! I read your topic and it sounds really interesting. :slightly_smiling_face:
